### PR TITLE
Safari 13.1 camera detection issue

### LINKED ISSUE #993: Blank page localbitcoins.com
Hi i'm having this issue too, is it resolved?  whe I try with my laptop
And when I try with my iphone last version safari and chrome  said
UNSUPPORTED BROWSER, restar tthe process with the last version....

iOS: 13.4.1
thanks

### PR DESCRIPTION
**Pull request by kopijunkie**
# Problem
Users on Safari 13.1 in the latest iOS version 13.4.1  and MacOSX 10.15.4 are unable to complete the SDK flow if host app's integration has `uploadFallback` disabled. Fixes #993 #994 

The device camera is not getting detected as the SDK calls `navigator.mediaDevices.enumerateDevices()` more than one time during the flow and Safari 13.1 has a [bug](https://bugs.webkit.org/show_bug.cgi?id=209580) where subsequent calls to `navigator.mediaDevices.enumerateDevices()` results in devices returning as type `audioinput` only.

# Solution
* Disable polling for camera support with `navigator.mediaDevices.enumerateDevices()` if user is on Safari 13.1
* For Safari 13.1 only, assume that if every device's `kind` includes "input" there is camera support 

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-996-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/utils/index.js [hunk 1]
```diff
@@ -26,6 +26,15 @@ export const preventDefaultOnClick = callback => event => {
 // https://stackoverflow.com/questions/58019463/how-to-detect-device-name-in-safari-on-ios-13-while-it-doesnt-show-the-correct
 const isIOS = (/iPad|iPhone|iPod/.test(navigator.platform || '') || (navigator.platform === 'MacIntel' && navigator.maxTouchPoints > 1)) && !window.MSStream
 
+// WARN: only intended to target Safari 13.1 which has a reported bug handling enumerateDevices()
+// https://bugs.webkit.org/show_bug.cgi?id=209580
+export const isSafari131 = () => {
```
Comment (josokinas) [inline src/components/utils/index.js:31]: [Minor] You execute this function multiple times during the app run. However, I believe the outcome would always be the same. I believe that better is to execute it once during the booting of an app, then store it as a variable and use it within components.

Comment (kopijunkie) [reply src/components/utils/index.js:31]: Good point. I was having a think about it last night & had the same thought.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-996-pr-onfido-sdk-ui-onfido.surge.sh

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,15 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
...
 ## [next-version]
 
+### Added
+
+### Changed
+- Internal: Remove unused dependencies and scripts from `package.json`
+
+### Fixed
+- UI: Accessibility - Focus is at document start
+- UI: Camera not detected on Safari 13.1 on iOS 13.4.1, MacOSX 10.15.4
```
Comment (josokinas) [inline CHANGELOG.md:17]: [Minor] OS naming:
- earlier: Mac OS X
- now: macOS

I believe you're referring to earlier version here.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-996-pr-onfido-sdk-ui-onfido.surge.sh

File: CHANGELOG.md (hunk 2 above)
Comment (kopijunkie) [reply CHANGELOG.md:17]: Ah yes. You're right!

**Comment by rfreitas** [comment]
Travis automatic deployment: https://996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://996-pr-onfido-sdk-ui-onfido.surge.sh

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,15 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
...
 ## [next-version]
 
+### Added
+
+### Changed
+- Internal: Remove unused dependencies and scripts from `package.json`
+
+### Fixed
+- UI: Accessibility - Focus is at document start
+- UI: Camera not detected on Safari 13.1 on iOS 13.4.1, macOS 10.15.4)
```
Comment (josokinas) [inline CHANGELOG.md:17]: I don't think you need closing `)` here 🙈

Comment (kopijunkie) [reply CHANGELOG.md:17]: Oops. You're right! 🙈

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-996-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://996-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**